### ISSUE TITLE
Prohibit changing the contents of the post and comments after a period.

### ISSUE THREAD

**Issue by leftside2** [HelloZeroNet/ZeroTalk#64]
Need to prohibit editing the title and contents of the post or comments once 5 minutes pass after the user uploaded it.

He has to remove the title or contents of it if wanting to remove his post or comment.

This is because some users edit their posts or comments too frequently even after the debate was over. As a result they twist or distort the original flow of the conversation so that they make new visitors to confuse or misunderstand the debate between the post author and the commenters.

http://127.0.0.1:43110/Me.ZeroNetwork.bit/?Profile/1SunAWK2VUT9GQK32MpwRfFPVgcBSJN9a/16dEowtNbV89NxPcDrAKkmnuXtiJVoVcDm/leftside@zeroid.bit

sos4sos@riseup.net 2018-03-07 18-14-10

**Comment by slrslr** [comment]
I want to continue being able to edit my posts anytime. I think it should be my right to edit/delete my content.

**Comment by leftside2** [comment]
@slrslr : Yeah, as you wish. But it would be great if we make a feature which the site owner could enable or disable.
